erpnext/patches/v13_0/create_leave_policy_assignment_based_on_employee_current_leave_policy.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2019, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+
+import frappe
+
+def execute():
+    employees_with_leave_policy = frappe.db.sql("SELECT name, leave_policy FROM `tabEmployee` WHERE leave_policy IS NOT NULL and leave_policy != ''", as_dict = 1)
```

Comment:
Run the patch only if leave_policy column exists in Employee and Grade table.

`if "leave_policy" not in frappe.db.get_table_columns("Employee"): return`

Because of this travis is failing